cmd/manager/main.go

```diff
@@ -20,7 +23,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
-var log = logf.Log.WithName("cmd")
+var log = logf.Log.WithName("ndm-operator cmd")
```

Comment:
_n: can this just be `ndm-operator`? Why is there a suffix of `cmd` ?

Reply:
Changed it to "ndm-operator"